Dockerfile

```diff
@@ -78,15 +78,15 @@ RUN dotnet publish Jellyfin.Server --disable-parallel --configuration Release --
 FROM app
 
 ENV HEALTHCHECK_URL=http://localhost:8096/health
+ENV JELLYFIN_CONFIG_DIR=/config
```

Comment:
JELLYFIN_DATA_DIR needs to be set instead, as you want to replace `--datadir`. JELLYFIN_CONFIG_DIR would replace the option `--configdir`.